packages/devtools_app/lib/src/code_size/file_import_container.dart

```diff
@@ -263,7 +268,12 @@ class _DualFileImportContainerState extends State<DualFileImportContainer> {
           children: [
             RaisedButton(
               onPressed: firstImportedFile != null && secondImportedFile != null
-                  ? () => widget.onAction(firstImportedFile, secondImportedFile)
+                  ? () => widget.onAction(
+                        firstImportedFile,
+                        secondImportedFile,
+                        (errorMessage) =>
```

Comment:
rename to error to be consistent with snapshot callback

Reply:
Done